tools/slurm_train.sh

```diff
@@ -21,4 +21,4 @@ srun -p ${PARTITION} \
     --cpus-per-task=${CPUS_PER_TASK} \
     --kill-on-bad-exit=1 \
     ${SRUN_ARGS} \
-    python -u tools/train.py ${CONFIG} --work-dir=${WORK_DIR} --launcher="slurm" ${PY_ARGS}
+    python -u $(dirname "$0")/train.py ${CONFIG} --work-dir=${WORK_DIR} --launcher="slurm" ${PY_ARGS}
```

Comment:
please modify this [line](https://github.com/JingweiZhang12/mmtracking/blob/trackingnet_alone/tools/analysis/mot/dist_mot_search.sh#L9).
It should be `mot_param_search.py` rather than `search.py`